Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,21 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - url: URL to save object to.
+    ///   - encoder: JSONEncoder.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public static func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder=JSONEncoder()) throws {
```

Comment:
This should not be a static function. Add spacing for the encoder param.
```suggestion
    public func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder = JSONEncoder()) throws {
```

Reply:
You should add the parameter
```Swift
attributes: [FileAttributeKey: Any]? = nil
```
and pass it to `createFile`